xml/System.Net.Http/HttpRequestMessage.xml

```diff
@@ -461,8 +461,29 @@
       </ReturnValue>
       <Docs>
         <summary>Gets or sets the HTTP message version.</summary>
-        <value>The HTTP message version. The default in the .NET Framework and earlier versions of .NET Core is 1.1. In .NET Core 2.1 and later, it is 2.0.</value>
-        <remarks>Starting with .NET  Core 2.1, the default value of the `Version` property changed from 1.1 to 2.0.</remarks>
+        <value>The HTTP message version. The default value is <c>1.1</c>, unless you're targeting .NET Core 2.1 or 2.2. In that case, the default value is <c>2.0</c>.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+          
+## Remarks
+
+The default message version differs based on the version of .NET Framework or .NET Core you're using. .NET Core 2.1 changed the default value `2.0`. In .NET Core 3.0 the default value was reverted back to `1.1`.
+
+### .NET Framework
+
+The default message version is `1.1`.
+
+### .NET Core
+
+The following table lists the default message version based on the version of .NET Core:
+
+| .NET Core version           | HTTP request version |
+| --------------------------- | -------------------- |
+| 1.0<br/>1.1<br/>2.0<br/>3.0 | `1.1`                |
+| 2.1<br/>2.2                 | `2.0`                |
```

Comment:
I'd either remove all the text starting at line 472 or make this future proof. Remembering to come back here for every release would be difficult.

```suggestion
| 2.1<br/>2.2                 | `2.0`                |
| Any other .NET Core version                 | `1.1`                |
```